src/extensions/default/CodeFolding/main.js

```diff
@@ -232,27 +224,30 @@ define(function (require, exports, module) {
       */
     function createGutter(editor) {
         var cm = editor._codeMirror;
-        if (!cm) {
-            return;
-        }
         var path = editor.document.file.fullPath, _lineFolds = prefs.getFolds(path);
         _lineFolds = _lineFolds || {};
         cm._lineFolds = _lineFolds;
         var gutters = cm.getOption("gutters").slice(0);
         var lnIndex = gutters.indexOf("CodeMirror-linenumbers");
-        gutters.splice(lnIndex + 1, 0, gutterName);
+        gutters.splice(lnIndex + 1, 0, GUTTER_NAME);
+        $(editor.getRootElement()).addClass("folding-enabled");
```

Comment:
This enables us to apply different padding to line numbers when the folding gutter is present vs. not.